src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -2796,11 +2811,13 @@ well as menu structures (for main menu and popup menus).
         context="import"
         windowMode="main"/>
 
-   <cmd id="importDatasetFromSAV"
-        label="Import Dataset from SAV..."
-        menuLabel="From _SAV..."
-        context="import"
-        windowMode="main"/>
+<!--   This has a duplicate ID to below.  Is that intentional?  Effect seems to be that below overwrites this command-->
```

Comment:
Good catch, thanks! I am fixing this separately, once done please merge back into your fork to avoid merge conflict.